src/wallet/wallet.cpp

```diff
@@ -1118,10 +1118,10 @@ void CWallet::MarkConflicted(const uint256& hashBlock, const uint256& hashTx)
     }
 }
 
-void CWallet::SyncTransaction(const CTransactionRef& ptx, const CBlockIndex *pindexBlockConnected, int posInBlock) {
+void CWallet::SyncTransaction(const CTransactionRef& ptx, const CBlockIndex *pindexBlock, int posInBlock) {
```

Comment:
because SyncTransaction is only ever called by a function which uses the variable name pindex, and only uses that variable to call another function that uses the variable name pindex (actually pIndex). Changing the variable name in the middle of the stack seems unnecessary. And since I'm already touching these lines, I might as well.

I should have renamed this pindex, not pindexBlock. Doing that now.